src/Commands/Env/CloneContentCommand.php

```diff
@@ -128,6 +145,7 @@ private function cloneDatabase(array $options)
         $params = [
             'clear_cache' => $options['cc'],
             'updatedb' => $options['updatedb'],
+            'wp_replace_siteurl' => $options['search-replace'] ?? "None",
```

Comment:
Not 100% sure on None vs "" but I expect you have answered this through testing.